content/robertstettner/drone-codecov/index.md

```diff
@@ -0,0 +1,55 @@
+---
+date: 2017-07-19T00:00:00+00:00
+title: Codecov
+author: robertstettner
+tags: [ codecov, test, coverage, reporting ]
+repo: robertstettner/drone-codecov
+logo: codecov.svg
+image: robertstettner/drone-codecov
+---
+
+This plugin allows for pushing test coverage results to Codecov. The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
+  codecov:
+    image: robertstettner/drone-codecov
+    token: ${CODECOV_TOKEN}
```

Comment:
You need to add `secret` section like that http://plugins.drone.io/appleboy/drone-gitlab-ci/